foundation/foundation-archetype/test-project-archetype.sh

```diff
@@ -175,7 +175,7 @@ cp ../../../src/test/resources/test-project-archetype-script/maven-build-cache-c
 echo -e "\n# maven-suppress-warnings" >> Tiltfile
 
 echo -e "\nINFO: Running full build to generate project structure\n"
```

Comment:
The instances below here need to be `./mvnw` since that is how the the instructions now work.